src/UserService.Business/Commands/Certificate/EditCertificateCommand.cs

```diff
@@ -23,111 +25,113 @@
 
 namespace LT.DigitalOffice.UserService.Business.Commands.Certificate
 {
-    public class EditCertificateCommand : IEditCertificateCommand
+  public class EditCertificateCommand : IEditCertificateCommand
+  {
+    private readonly IAccessValidator _accessValidator;
+    private readonly IHttpContextAccessor _httpContextAccessor;
+    private readonly IUserRepository _userRepository;
+    private readonly ICertificateRepository _certificateRepository;
+    private readonly IPatchDbUserCertificateMapper _mapper;
+    private readonly ICreateImageDataMapper _createImageDataMapper;
+    private readonly IRequestClient<ICreateImagesRequest> _rcImage;
+    private readonly ILogger<EditCertificateCommand> _logger;
+
+    private Guid? GetImageId(AddImageRequest addImageRequest, List<string> errors)
     {
-        private readonly IAccessValidator _accessValidator;
-        private readonly IHttpContextAccessor _httpContextAccessor;
-        private readonly IUserRepository _userRepository;
-        private readonly ICertificateRepository _certificateRepository;
-        private readonly IPatchDbUserCertificateMapper _mapper;
-        private readonly IRequestClient<IAddImageRequest> _rcImage;
-        private readonly ILogger<EditCertificateCommand> _logger;
-
-        private Guid? GetImageId(AddImageRequest addImageRequest, List<string> errors)
+      Guid? imageId = null;
+
+      if (addImageRequest == null)
+      {
+        return null;
+      }
+
+      Guid userId = _httpContextAccessor.HttpContext.GetUserId();
+
+      const string errorMessage = "Can not add certificate image to certificate. Please try again later.";
+
+      try
+      {
+        Response<IOperationResult<Guid>> response = _rcImage.GetResponse<IOperationResult<Guid>>(
+          ICreateImagesRequest.CreateObj(_createImageDataMapper.Map(
+            new List<AddImageRequest>() { addImageRequest }, userId),ImageSource.User), default, TimeSpan.FromSeconds(5)).Result;
```

Comment:
add space before enum in arguments or better move to next line arguments (each arg of create obj in separate lines)